public/index.html

```diff
@@ -1267,6 +1267,37 @@ <h4 class="wide100p textAlignCenter">
                                             </div>
                                         </div>
                                     </div>
+                                    <div data-newbie-hidden data-tg-type="mancer, ooba, koboldcpp, aphrodite, tabby" name="dryBlock" class="wide100p">
+                                        <h4 class="wide100p textAlignCenter">
+                                            <label data-i18n="DRY Sampling">DRY Sampling</label>
+                                            <div class=" fa-solid fa-circle-info opacity50p " data-i18n="[title]DRY Sampling" title="Allows you to apply the DRY repetition penalty as outlined in https://github.com/oobabooga/text-generation-webui/pull/5677. Set multiplier to 0 to disable"></div>
+                                        </h4>
+                                        <div data-tg-type="mancer, ooba, koboldcpp, aphrodite, tabby" class="flex-container flexFlowRow gap10px flexShrink">
```

Comment:
data-tg-type is not needed in nested blocks - it was already hidden by the top-level block. Feel free to remove here (and two other instances below)